prisma/schema.prisma

```diff
@@ -7,18 +7,24 @@ datasource db {
     url      = env("DATABASE_URL")
 }
 
+enum Role {
+    USER
+    ADMIN
+}
```

Comment:
Siirsin `enum`:in `model`ien yläpuolelle? Onko tämä ihan ok tapa, vai onko parempi että on `model`ien alapuolella+

Reply:
`model`ien yläpuolella on parempi 👍